lib/playgroundbook_renderer/chapter_collator.rb

```diff
@@ -11,47 +11,29 @@ def initialize(page_writer = PageWriter.new, ui = Cork::Board.new)
       @ui = ui
     end
 
-    def collate!(chapter_name, chapter_file_contents, imports)
+    def collate!(chapter_name, parsed_chapter, imports)
```

Comment:
I wasn't sure on that, thanks for clarifying. They're mutating in that they mutate the filesystem, but that's it.